internal/webhook/stage/webhook.go

```diff
@@ -127,10 +127,10 @@ func (w *webhook) Default(ctx context.Context, obj runtime.Object) error {
 			}
 		}
 
-		if verReq, ok := kargoapi.AbortAnnotationValue(stage.Annotations); ok {
+		if verReq, ok := kargoapi.AbortVerificationAnnotationValue(stage.Annotations); ok {
 			var oldVerReq *kargoapi.VerificationRequest
 			if oldStage != nil {
-				oldVerReq, _ = kargoapi.AbortAnnotationValue(oldStage.Annotations)
+				oldVerReq, _ = kargoapi.AbortVerificationAnnotationValue(oldStage.Annotations)
 			}
 			// If the abort request has changed, enrich the annotation with the
 			// actor and control plane information.
```

Comment:
Do you suppose we ought to do this for the Promotions webhook also?

Reply:
Yes, great catch.

Reply:
Addressed in cd45c93de0133d93d42d384e87899d264593c72c.